src/format/mod.rs

```diff
@@ -38,9 +38,19 @@ use offset::{FixedOffset, Offset};
 use {Datelike, Timelike};
 use {ParseWeekdayError, Weekday};
 
+#[cfg(all(feature = "locales", any(feature = "alloc", feature = "std", test)))]
+pub(crate) mod locales;
+
 pub use self::parse::parse;
 pub use self::parsed::Parsed;
 pub use self::strftime::StrftimeItems;
+#[cfg(all(feature = "locales", any(feature = "alloc", feature = "std", test)))]
+pub use pure_rust_locales::Locale;
+
+#[cfg(not(feature = "locales"))]
+#[allow(dead_code)]
```

Comment:
why is this `allow(dead_code)` necessary?

Reply:
No. It's probably with the long `#[cfg]` I had that there was an issue

Reply:
done